### PR TITLE
#32 social impact cooperative section

### PR DESCRIPTION
**Pull request by Eddahkarambu**
# [#32 ] Social Impact Cooperative Section 

<p> This is a section that describes the organizations' common objectives </p>

## Changes 
<p> The component displays by reusing the banners feature from <em>#31</em>

## Desktop Preview 
WhatsApp Image 2023-02-16 at 10 28 43 (1)

## Mobile Preview 
WhatsApp Image 2023-02-16 at 10 28 43

## Contributors 
<h2> Code was written by @Essyuge ,  @JohnstoneDev & I . 

<h4> @IanOdhiambo9 Kindly Review. </h4>

### CODE REVIEW

File: libs/pages/elewa/home/src/lib/features-elewa-home.module.ts [hunk 1]
```diff
@@ -16,24 +20,37 @@ import { NextDirective } from './directives/next.directive';
...
+    HomeHeroSectionComponent,
+    HomeJobsSectionComponent,
+    SocialImpactComponent,
     TeamAndPartnersComponent,
     ActivitySectionComponent,
     NextDirective,
     PrevDirective
   ],
-  exports: [HomePageComponent, HomeHeroSectionComponent, HomeJobsSectionComponent, TeamAndPartnersComponent, ActivitySectionComponent, NextDirective, PrevDirective],
+  // exports: [HomePageComponent, HomeHeroSectionComponent, HomeJobsSectionComponent, TeamAndPartnersComponent, ActivitySectionComponent, NextDirective, PrevDirective],
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/features-elewa-home.module.ts:54]: ?? duplication

File: libs/pages/elewa/home/src/lib/social-impact/social-impact.component.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<elewa-group-elewa-group-image-and-text-banner
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/social-impact/social-impact.component.html:1]: this should be in a components folder

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Rename social impact to cooperative section and move it into a components folder. Really clean design. Good job so far 👍

**Comment by Eddahkarambu** [comment]
@LemmyMwaura Kindly review.

**Comment by IanOdhiambo9** [comment]
I'm closing this PR because it contains package.json and package-lock.json files.

Please do a New PR without this files.

**Pull request closed without merging**